client/components/EventCard.js

```diff
@@ -113,9 +113,22 @@ class EventCard extends React.Component {
 
   @autobind
   async deleteEvent() {
-    const response = await fetch(`/api/events/${this.state.event._id}`, {
-      credentials: 'same-origin', method: 'DELETE',
-    });
+    const event = JSON.parse(JSON.stringify(this.props.event));
+    const observerEvent = jsonpatch.observe(event);
+    event.active = false;
+    const patches = jsonpatch.generate(observerEvent);
+    const response =  await fetch(
+      `/api/events/${event._id}`,
+      {
+        headers: {
+          Accept: 'application/json',
+          'Content-Type': 'application/json',
+        },
+        method: 'PATCH',
```

Comment:
No. its suppose to set as active=false. At the old code the api understand a delete method as changing this field